lib/take_away.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'menu'
+class Takeaway
+
+  attr_reader :my_order, :check_out
+  
+  MENU = { "hotdog"=>1.99,
+          "cheesebuger"=>1.49,
+          "Hamburger"=>0.99,
+          "Fries"=>0.89, }
+
+  def initialize(menu_klass = Menu.new)
+    @menu = menu_klass
+    @my_order = {}
+    @total = 0.0
+  end
+
+  def read_menu
+    MENU
+  end
+
+  def order(menu_item)
+    if my_order.has_key?(menu_item)
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`. (https://github.com/bbatsov/ruby-style-guide#hash-key)